.scripts/remove_duplicates.py

```diff
@@ -0,0 +1,150 @@
+import pandas as pd
+import os
+import ast
+import sys
+
+os.makedirs('raw_statements_cleaned', exist_ok=True)
+
+# define any necessary paths
+observable_translated_statements = ['raw_statements/observable_gpt4o_ar.csv',
+                        'raw_statements/observable_gpt4o_bn.csv',
+                        'raw_statements/observable_gpt4o_es.csv',
+                        'raw_statements/observable_gpt4o_fr.csv',
+                        'raw_statements/observable_gpt4o_hi.csv',
+                        'raw_statements/observable_gpt4o_ja.csv',
+                        'raw_statements/observable_gpt4o_pt.csv',
+                        'raw_statements/observable_gpt4o_ru.csv',
```

Comment:
This is fine but let's keep track of our languages and I prefer a f string with list of languages for things like this
```python
f"raw_statements/observable_gpt4o_{language}.csv"
```